maker/src/actor_system.rs

```diff
@@ -77,6 +75,7 @@ pub struct ActorSystem<O: 'static, W: 'static> {
             cfd::RatesChannel,
         >,
     >,
+    _oracle_actor: Address<O>,
```

Comment:
❓Is this now necessary because we don't keep a reference to this actor in `cfd_actor` anymore?

Reply:
I never fully understood the underlying purpose of these references. I get that they are to keep the actor alive, but why must it be done in this way? Couldn't it be done "for free" by keeping strong references in whatever part of the application provides the input messages to these actors?